docs/fpc-registration-old.puml

```diff
@@ -0,0 +1,83 @@
+@startuml
+
+'- force wrapping of longer text
+'skinparam wrapWidth 400
+
+!pragma teoz true
+
+
+title Fabric Private Chaincode - Enclave registry - Current approach
+
+actor	Admin order 10
+participant ECC order 20
+participant ERCC order 40
+entity IAS order 50
+collections Orderer order 70
+collections Peers order 80
+
+note right of Peers
+  **Drawbacks**
+  - Custom validation at __every__ peer
+  - Single-endorser policy only
```

Comment:
also note a single-endorser policy doesn't really make sense security-wise for these kind of peer-endorsed lifecycle/upgrade events (differently to attestation-based endorsement where a single one is fine as we trust the TEE fully, at least for now ...)